### PR TITLE
Hide spark config

### LINKED ISSUE #803: Hide spark config in Using Dask and Koalas Guide
In #801 we had to move the spark config from a raw rst cell that was using `:suppress:` to hide the config to a normal cell, which isn't useful for users reading the guide to understand how to use Koalas or Dask in Woodwork. If possible, we should find a way to hide this config from users that doesn't break our build-docs test.

Before the change, the docs build was failing in the CI run (not locally, though) when you pushed new commits. It would pass, however, when rerunning the job. In featuretools, the cell is being suppressed successfully (so far), but the guide is in the format of an `rst` file instead of `ipynb`, so maybe there's some difference in execution there.

### PR DESCRIPTION
**Pull request by simha104**
- added "nbsphinx": "hidden" to cell's metadata
- Closes #803

-----
*After creating the pull request: in order to pass the **release_notes_updated** check you will need to update the "Future Release" section of* `docs/source/release_notes.rst` *to include this pull request.*

### CODE REVIEW

File: docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb [hunk 1]
```diff
@@ -1,346 +1,348 @@
...
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.8.2"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 4
+    "cells": [
+        {
```
Comment (tamargrey) [inline docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb:3]: Seems like the indentation got changed when you added the entry to the metadata, so the diff is showing the whole file and not just the small change. Was this an automatic formatting thing?

Comment (tamargrey) [reply docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb:3]: It doesn't seem to be impacting the docs build, but probably best to be able to review just the diff of interest.

Comment (simha104) [reply docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb:3]: Yep it's an automatic formatting thing

Comment (simha104) [reply docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb:3]: Is there a way I can save without formatting just once?

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -9,6 +9,7 @@ Future Release
     * Fixes
     * Changes
         * Update ``Datetime.transform`` to use default nrows value when calling ``_infer_datetime_format`` (:pr:`1137`)
+        * Hide spark config in Using Dask and Koalas Guide (:pr:`1139`)
```
Comment (tamargrey) [inline docs/source/release_notes.rst:12]: This should go under the documentation changes section!

Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:12]: Also - don't forget to add yourself to the list of contributors below!

**Review by thehomebrewnerd — APPROVED** [review]

**Review by tamargrey — APPROVED** [review]
lgtm!

**Pull request merged**